eo-parser/src/main/resources/org/eolang/parser/move-versions-deeper.xsl

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+The MIT License (MIT)
+
+Copyright (c) 2016-2023 Objectionary.com
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" id="move-versions-deeper" version="2.0">
+  <!--
+  Here we move "ver" attribute from the top object deeper to pre-last one (before <o base="Q"/>)
+
+  1. The reason of transformation - move version to pre-last object ("org"). Then on "transpile"
+     step "to-java.xsl" will add extra {@code attr("some-hash).get()} java code.
+     More details here: https://github.com/objectionary/eo/issues/2503
```

Comment:
@maxonfjvipon I tried to find the reason for the `move-versions-deeper.xsl` transformation in the [issue](https://github.com/objectionary/eo/issues/2503) that you linked and I've found only a high-level description of the problem, not the reason for that transformation. I still can only guess why we might need it. Moreover:
> This transformation should be applied in the end, after all other transformations are done.

doesn't explain the reason and It seems contradictory with the (4) point:

> Because of point 3 this transformation should be applied many times, until it does not affect xml anymore

Reply:
@volodya-lombrozo I extended the doc, please have a look